### PR TITLE
Remove `chrono`

### PR DESCRIPTION
**Pull request by Adam-Gleave**
# Description of change

Removes the `chrono` crate.

I have also created a new `time` module in `bee-common` for getting local time and creating formatted strings, since the same code is used in a couple of different places.

## Links to any relevant issues

#783 and #779.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Run the node to ensure timestamps format correctly. The output shows as:

```
2021-10-19 13:18:54 bee_protocol::workers::message::payload    INFO  Running.
```
This is in the correct local time to me (UTC +1).

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that new and existing unit tests pass locally with my changes
- [x] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

**Comment by Adam-Gleave** [comment]
Adding to #786.

File: bee-common/bee-common/src/logger/mod.rs [hunk 1]
```diff
@@ -7,6 +7,9 @@ mod config;
 
 pub use config::{LoggerConfig, LoggerConfigBuilder, LoggerOutputConfig, LoggerOutputConfigBuilder};
 
+#[cfg(not(feature = "tokio-console"))]
+use crate::time as time_util;
```
Comment (thibault-martinez) [inline bee-common/bee-common/src/logger/mod.rs]: Can you maybe just use `crate::time::format` directly in the macro and not import it ?

File: bee-common/bee-common/src/time.rs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// Copyright 2020-2021 IOTA Stiftung
```
Comment (thibault-martinez) [inline bee-common/bee-common/src/time.rs]:
```suggestion
// Copyright 2021 IOTA Stiftung
```

**Review by thibault-martinez — CHANGES_REQUESTED** [review]

File: bee-common/bee-common/src/logger/mod.rs [hunk 3]
```diff
@@ -27,9 +27,11 @@ pub enum Error {
 #[cfg(not(feature = "tokio-console"))]
 macro_rules! log_format {
     ($target:expr, $level:expr, $message:expr, $target_width:expr, $level_width:expr) => {
+        use crate::time as time_util;
```
Comment (thibault-martinez) [inline bee-common/bee-common/src/logger/mod.rs]: Sorry I meant `crate::time::format` directly, without previously `use`ing it

Comment (Adam-Gleave) [reply bee-common/bee-common/src/logger/mod.rs]: We'll have to import `crate::time::now_local` too, which seems ugly to me 😆

Comment (thibault-martinez) [reply bee-common/bee-common/src/logger/mod.rs]: I don't particularly like it either but it's kinda convention to fully qualify paths inside macros... We also do that in other macros in the codebase

Comment (thibault-martinez) [reply bee-common/bee-common/src/logger/mod.rs]: Feel free to keep the conditional use if you really prefer it ofc, just suggesting :)

Comment (Adam-Gleave) [reply bee-common/bee-common/src/logger/mod.rs]: Ah sure, didn't realise that!

File: bee-ledger/src/workers/snapshot/worker.rs [hunk 4]
```diff
@@ -48,7 +48,7 @@ where
 
             info!(
                 "Loaded snapshot from {} with snapshot index {}, entry point index {} and pruning index {}.",
-                Utc.timestamp(info.timestamp() as i64, 0).format("%d-%m-%Y %H:%M:%S"),
+                time::format(&time::from_unix_timestamp(info.timestamp() as i64)),
```
Comment (thibault-martinez) [inline bee-ledger/src/workers/snapshot/worker.rs]: Maybe a function `time::format_unix_timestamp` or similar as it seems like a frequent use case ?

Comment (Adam-Gleave) [reply bee-ledger/src/workers/snapshot/worker.rs]: Yeah, I think that might be helpful -- I'll add it.

Comment (thibault-martinez) [inline bee-ledger/src/workers/snapshot/worker.rs]: Use the newly created function ?

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**